tests/protocol/test_wire.py

```diff
@@ -209,26 +209,28 @@ def test_get_host_ga_plugin(self, *args):
             self.assertEqual(goal_state.container_id, host_plugin.container_id)
             self.assertEqual(goal_state.role_config_name, host_plugin.role_config_name)
 
-    @skip_if_predicate_true(lambda: True, "Needs to be re-enabled before release 2.2.47")
-    def test_upload_status_blob_default(self, *args):
-        """
-        Default status blob method is HostPlugin.
-        """
-        with create_mock_protocol(status_upload_blob=testurl, status_upload_blob_type=testtype) as protocol:
-            protocol.client.status_blob.vm_status = VMStatus(message="Ready", status="Ready")
+    def test_upload_status_blob_should_use_the_host_channel_by_default(self, *_):
```

Comment:
The changes below are the meat of this PR. Now we mock the interaction with the host/storage in terms of http requests, instead of reaching into implementation internals.